Scripts/UI/PinnedToolButton.cs

```diff
@@ -1,4 +1,7 @@
-﻿using System;
+﻿#if !UNITY_EDITORVR
+#pragma warning disable 67, 414, 649
```

Comment:
Same here, with selected. In fact, it selected is _actually_ unused. It's just not reported as such because it's a public field on a public class. Anyway when I remove selected and the warning disable, I don't get any warnings from this class